src/strategy/dungeons/wotlk/forgeofsouls/FosMultipliers.cpp

```diff
@@ -0,0 +1,55 @@
+#include "FosMultipliers.h"
+#include "FosActions.h"
+#include "GenericSpellActions.h"
+#include "ChooseTargetActions.h"
+#include "MovementActions.h"
+#include "FosTriggers.h"
+#include "FosActions.h"
+
+
+float BronjahmMultiplier::GetValue(Action* action) {
+    Unit* boss = nullptr;
```

Comment:
Don't need to pre-init pointer here:
`Unit* boss = AI_VALUE2(Unit*, "find target", "bronjahm");`

Will return nullptr if target not found